src/plot_api/plot_api.js

```diff
@@ -312,7 +312,7 @@ Plotly.plot = function(gd, data, layout, config) {
 
         // show annotations and shapes
         Registry.getComponentMethod('shapes', 'draw')(gd);
-        Registry.getComponentMethod('annoations', 'draw')(gd);
```

Comment:
The second `Annotations.draw` call was added in https://github.com/plotly/plotly.js/pull/243 [here](https://github.com/plotly/plotly.js/pull/243/files?diff=unified#diff-2941ab69a12080c0633ff4ac8ea3aa83R306).